app/src/main/kotlin/io/homeassistant/companion/android/controls/HaControlInfo.kt

```diff
@@ -9,5 +9,6 @@ data class HaControlInfo(
     val serverName: String? = null,
     val area: AreaRegistryResponse? = null,
     val authRequired: Boolean = false,
-    val baseUrl: String? = null
+    val baseUrl: String? = null,
+    val splitMultiServerIntoStructure: Boolean
```

Comment:
You should probably also add a default value, to follow the design of this class.